example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+
+    <application
+
+        android:usesCleartextTraffic="true"
+        tools:targetApi="28"
+        tools:ignore="GoogleAppIndexingWarning">
+
+        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" android:exported="false" />
```

Comment:
What is this for `DevSettingsActivity`?

Reply:
It was generated by the template from react native module, I think it's normal to have it 
you can check [there 
](https://reactnative.dev/docs/0.63/integration-with-existing-apps#configuring-permissions)